base_contact/i18n/fr.po

```diff
@@ -0,0 +1,193 @@
+"Last-Translator: EL Hadji DEM <elhadji.dem@savoirfairelinux.com>\n"
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: \n"
+"X-Generator: Poedit 1.5.4\n"
+
+#. module: base_contact
+#: view:res.partner:0
+msgid "City"
+msgstr "Ville"
+
+#. module: base_contact
+#: view:res.partner:0
+msgid "other position"
+msgstr "fonction"
+
+#. module: base_contact
+#: view:res.partner:0
+msgid "Contacts"
+msgstr "Contacts"
+
+#. module: base_contact
+#: view:res.partner:0
+msgid ""
+"To see personal information about this contact, please go to to the his "
+"person form:"
+msgstr ""
+"Pour voir des informations personnelles sur ce contact, s'il vous plaît "
+"aller au formulaire de la personne:"
+
+#. module: base_contact
+#: view:res.partner:0
+msgid "Street..."
+msgstr "Rue..."
+
+#. module: base_contact
+#: view:res.partner:0
+msgid "State"
+msgstr "État"
+
+#. module: base_contact
+#: view:res.partner:0
+msgid "at"
+msgstr "à"
+
+#. module: base_contact
+#: view:res.partner:0
+msgid "Tags..."
+msgstr "Étiquettes..."
+
+#. module: base_contact
+#: view:res.partner:0
+msgid "Other Positions"
+msgstr "Fonctions"
+
+#. module: base_contact
+#: view:res.partner:0
+msgid "Phone:"
+msgstr "Téléphone:"
+
+#. module: base_contact
+#: view:res.partner:0
+msgid "Company"
+msgstr "Organisme"
+
+#. module: base_contact
+#: field:res.partner,contact_id:0
+msgid "Main Contact"
+msgstr "Principal contact"
+
+#. module: base_contact
+#: view:res.partner:0
+msgid "Fax:"
+msgstr "Fax :"
+
+#. module: base_contact
+#: code:addons/base_contact/base_contact.py:31
+#, python-format
+msgid "Standalone Contact"
+msgstr "Contact autonome"
+
+#. module: base_contact
+#: help:res.partner,nationality_id:0
+msgid "Nationality."
+msgstr "Nationalité."
+
+#. module: base_contact
+#: view:res.partner:0
+msgid "Address"
+msgstr "Adresse"
+
+#. module: base_contact
+#: field:res.partner,nationality_id:0
+msgid "Nationality"
+msgstr "Nationalité"
+
+#. module: base_contact
+#: code:addons/base_contact/base_contact.py:32
+#, python-format
+msgid "Attached to existing Contact"
+msgstr "Contact rattaché à un contact existant"
+
+#. module: base_contact
+#: view:res.partner:0
+msgid "ZIP"
+msgstr "Code postal"
+
+#. module: base_contact
+#: view:res.partner:0
+msgid "Country"
+msgstr "Pays"
+
+#. module: base_contact
+#: field:res.partner,birthdate_date:0
+msgid "Birthdate"
+msgstr "Date de naissance"
+
+#. module: base_contact
+#: view:res.partner:0
+msgid "Person"
+msgstr "Personne"
+
+#. module: base_contact
+#: view:res.partner:0
+msgid "Contact"
+msgstr "Contact"
+
+#. module: base_contact
+#: view:res.partner:0
+msgid "Mobile:"
+msgstr "Portable :"
+
+#. module: base_contact
+#: view:res.partner:0
+msgid "All partner positions"
+msgstr "Tous les contacts"
+
+#. module: base_contact
+#: view:res.partner:0
+msgid "{'invisible': [('contact_type','=','attached')]}"
```

Comment:
Better remove this line that can be conflictive (it's a bug in the translation export)